packages/create-liveblocks-app/scripts/demo.js

```diff
@@ -45,7 +43,7 @@ fs.rmSync(testDir, { recursive: true, force: true });
 fs.mkdirSync(testDir);
 fs.writeFileSync(path.join(testDir, "package.json"), pkgJson);
 
-execSync(`node ${esbuildOptions.outfile} ${flags}`, {
+execSync(`node ${esbuildOptions.outfile} ${process.argv.slice(2)}`, {
```

Comment:
What's happening here? This feels a bit brittle.

Reply:
This is just a demo script to test the built CLI locally, `npm run demo` etc. This change forwards all flags (the two first args are `node` and the file path) so we can do `npm run demo --upgrade` with it.